From: Jon Derrick <jonathan.derrick@intel.com>
Date: Tue, 20 Dec 2016 14:38:14 -0700
Subject: genhd: remove dead and duplicated scsi code
References: bsc#1020989,FATE#322379
Git-commit: 72c5296f9d64d8f5f27c2133e5f108a45a353d71
Patch-Mainline: v4.10-rc3

blk_scsi_cmd_filter use was deprecated by 4beab5c6 and the SCSI macros
are duplicated in blkdev.h, both likely reintroduced by a bad merge from
540eed56.

Signed-off-by: Jon Derrick <jonathan.derrick@intel.com>
Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jens Axboe <axboe@fb.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 include/linux/genhd.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/include/linux/genhd.h b/include/linux/genhd.h
index 2e80f4a..3c62da1 100644
--- a/include/linux/genhd.h
+++ b/include/linux/genhd.h
@@ -145,15 +145,6 @@ enum {
 	DISK_EVENT_EJECT_REQUEST		= 1 << 1, /* eject requested */
 };
 
-#define BLK_SCSI_MAX_CMDS	(256)
-#define BLK_SCSI_CMD_PER_LONG	(BLK_SCSI_MAX_CMDS / (sizeof(long) * 8))
-
-struct blk_scsi_cmd_filter {
-	unsigned long read_ok[BLK_SCSI_CMD_PER_LONG];
-	unsigned long write_ok[BLK_SCSI_CMD_PER_LONG];
-	struct kobject kobj;
-};
-
 struct disk_part_tbl {
 	struct rcu_head rcu_head;
 	int len;
-- 
1.8.5.6

